test/examples/greet.m

```diff
@@ -1,7 +1,7 @@
 function greeting = greet(user)
 % Returns a greeting.
 %
-% >> greet World
+% >> disp(greet('World'))
```

Comment:
I think I prefer to leave this as `>> greet World` and change the function to `function greet(user) ... disp(['Hello, ' user '!']);`

Can you also do this in README.md?

Reply:
otherwise, this looks good to me!

Reply:
Done with 63c7d9f3afeb39df6fc1765baef0eb16cf68088e.